cargo/spec/dependabot/cargo/file_fetcher_spec.rb

```diff
@@ -395,6 +395,13 @@
         stub_request(:get, url + "lib/sub_crate/Cargo.toml?ref=sha").
           with(headers: { "Authorization" => "token token" }).
           to_return(status: 404, headers: json_header)
+        # additional requests due to submodule searching
```

Comment:
I noticed that these tests are nested under the "workspace dependency" context. Does the path override only apply to cargo workspaces?

I might be misunderstanding what workspaces are for this ecosystem though if they differ from how npm implements them

Reply:
Yes I think once there are multiple Cargo.tomls involved it's a "workspace" but nothing else really changes, whereas with npm it looks like a workspace has to explicitly be created with `npm init -w`?